src/components/layers/LayerByValueList.svelte

```diff
@@ -0,0 +1,115 @@
+<script lang="ts">
+    /*
+        This layer uses dynamic values from a list of values.
+        The values per feature should be of the type 
+        PrefixKey|value
+
+        For example
+        Test|100
+        Test|200
+        Test|300
+
+        This connects to a slider and then will use the maximum value
+        less than the slider value.
+
+        Currently, the value is used to determine opacity of the feature outline
+    */
+
+    import { onMount, onDestroy } from 'svelte';
+    import { Writable } from "svelte/store"
+    import chroma from "chroma-js";
+    import L from 'leaflet';
+
+	import GeoJsonLayer from './GeoJsonLayer.svelte';
+    import { GetOrCreateControlStore } from '../../stores';
+    import type { Layer, LayerProperty, LayerByValueListArgs } from '../config';
+
+    export let id: string;
+    export let property: LayerProperty;
+    export let args: LayerByValueListArgs;
+
+    // these are things we can only load once mounted
+    let layer: L.GeoJSON;
+    let store: Writable;
+    let keys: string[] = [];
+    let key: string;
+    let unsubscribe: (() => void);
+
+    // FIXME types
+    let colorChroma: any;
+
+    onMount(() => {
+        // FIXME hard-coded to connect to a slider.
+        const id = `${property.id}.slider`;
+        args = property.args;
+        store = GetOrCreateControlStore(id);
```

Comment:
Why not pass args.initialValue in here?

Reply:
In the store? I am still figuring out life cycle. If two components call the store, who reads it first? I am not sure. What if the other place uses an initial value? I don't have a good answer for these yet, so I didn't include that fix in this PR.

One PR for one fix. This creates controls. It does not fix the store. That will require more work, more research, more testing, more code and more time.